packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDescriptionModal.tsx

```diff
@@ -0,0 +1,105 @@
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_INPUT_TYPE,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+  OdsInputValueChangeEventDetail,
+  OsdsInputCustomEvent,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsButton,
+  OsdsFormField,
+  OsdsInput,
+  OsdsModal,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface EditModalProps {
+  organizationDescription: string;
+  onCloseModal: () => void;
+  onEdit: (desc: string) => void;
+}
+
+const validateDescription = (desc: string) => /^.{1,256}$/.test(desc);
+
+export const EditDescriptionModal = ({
+  organizationDescription,
+  onCloseModal,
+  onEdit,
+}: EditModalProps) => {
+  const { t } = useTranslation('dashboard');
+  const [newDesc, setNewDesc] = useState<string>(organizationDescription || '');
+  const isValidDesc = validateDescription(newDesc);
+  const isButtonEnabled = isValidDesc && newDesc !== organizationDescription;
+
+  const handleSubmit = () => {
+    if (isValidDesc) {
+      onEdit(newDesc);
+      onCloseModal();
+    }
+  };
+
+  return (
+    <OsdsModal
+      color={ODS_THEME_COLOR_INTENT.info}
+      onOdsModalClose={onCloseModal}
+      dismissible
+    >
+      <OsdsText
+        color={ODS_THEME_COLOR_INTENT.text}
+        level={ODS_TEXT_LEVEL.heading}
+        size={ODS_TEXT_SIZE._400}
+      >
+        {t('managed_vcd_dashboard_edit_desc_modal_title')}
+      </OsdsText>
+
+      <OsdsFormField>
+        <OsdsText color={ODS_THEME_COLOR_INTENT.text} className="mt-6">
+          {t('managed_vcd_dashboard_edit_desc_modal_label')}
+        </OsdsText>
```

Comment:
you should be able to use the `label` slot
`<osds-text slot='label' color='text'>Label</osds-text>`